test/format/format.i

```diff
@@ -235,6 +252,49 @@ table_select_type(table_type type)
     return (tables[i]);
 }
 
+/*
+ * wiredtiger_open_session --
```

Comment:
I see that the naming convention wasn't introduced by this change, but calling these functions `wiredtiger_XXX` confuses me greatly. When I saw code calling this I wondered why we had added a special new entry point to the WiredTiger library that bypassed the regular open_session call. Could we replace this prefix with `wt_wrap_XXX` or something similar instead?